src/apps/frontend/components/input/password-input.tsx

```diff
@@ -7,7 +7,7 @@ import Input from '.';
 
 interface PasswordInputProps
   extends React.InputHTMLAttributes<HTMLInputElement> {
-  error: string;
+  error: string | false | undefined;
```

Comment:
Why would this be false?

This should be `error?: string` or `error: Nullable<string>`